rpc_jobs/rpc_aio.yml

```diff
@@ -169,6 +169,7 @@
 
     dsl: |
       // CIT Slave node
+      currentBuild.result = 'SUCCESS'
```

Comment:
That makes sense. Does that mean `delPubCloudSlave` failures could cause an inconsistency? Are you counting on the likelihood of failure there being low?